app/views/contacts/_form.html.erb

```diff
@@ -1,161 +1,6 @@
-
-
-    <fieldset class="field-section field-section--two-cols">
-      <legend><h2 class="field-section__title">Any dietary requirements?</h2></legend>
-
-      <div class="radio">
-        <%= form.radio_button :any_dietary_requirements, true, :id => "any_dietary_requirements_true" %>
-        <label for="any_dietary_requirements_true" class="radio__label">Yes</label>
-      </div>
-      <div class="radio">
-        <%= form.radio_button :any_dietary_requirements, false, :id => "any_dietary_requirements_false" %>
-        <label for="any_dietary_requirements_false" class="radio__label">No</label>
-      </div>
-
-      <div class="field field--span-two-cols">
-        <%= form.label :dietary_details, class: "field__label" %>
-        <%= form.text_area :dietary_details %>
-      </div>
-
-      <legend><h2 class="field-section__title">What cooking facilities do they have?</h2></legend>
-      <div class="field field--span-two-cols">
-        <%= form.text_area :cooking_facilities %>
-      </div>
-
-    </fieldset>
-
-    <fieldset class="field-section field-section--two-cols">
-      <legend><h2 class="field-section__title">What should we know about them?</h2></legend>
-
-      <div class="field field--span-two-cols">
-        <%= form.label :additional_info, class: "field__label" %>
-        <%= form.text_area :additional_info %>
-      </div>
-
-      <legend><h2 class="field-section__title">Are they eligible for free prescriptions?</h2></legend>
-      <div class="radio">
-        <%= form.radio_button :eligible_for_free_prescriptions, true, :id => "eligible_for_free_prescriptions_true" %>
-        <label for="eligible_for_free_prescriptions_true" class="radio__label">Yes</label>
-      </div>
-      <div class="radio">
-        <%= form.radio_button :eligible_for_free_prescriptions, false, :id => "eligible_for_free_prescriptions_false" %>
-        <label for="eligible_for_free_prescriptions_false" class="radio__label">No</label>
-      </div>
-
-    </fieldset>
-
-    <fieldset class="field-section">
-      <legend><h2 class="field-section__title">Is this person particularly vulnerable or high priority?</h2></legend>
-
-      <div class="notice">
-        <p>By <strong>vulnerable</strong> we mean:</p>
-        <ul>
-          <li>Over 70 years old</li>
-          <li>Under 70 years old and has an underlying health condition (i.e. normally asked by the NHS to get a flu jab)</li>
-          <li>Pregnant</li>
-        </ul>
-      </div>
-
-      <div class="radio">
-        <%= form.radio_button :is_vulnerable, true, :id => "is_vulnerable_true" %>
-        <label for="is_vulnerable_true" class="radio__label">Yes</label>
-      </div>
-      <div class="radio">
-        <%= form.radio_button :is_vulnerable, false, :id => "is_vulnerable_false" %>
-        <label for="is_vulnerable_false" class="radio__label">No</label>
-      </div>
-
-    </fieldset>
-
-
+    <%= render partial: 'form_fields', locals: { contact: @contact, form: form } %>
```

Comment:
This is copied verbatim, no changes.  The purpose is to be able to embed the fields from the contact form inside the combined needs/profile edit form